### PR TITLE
update project Page and adding table component

### PR DESCRIPTION
**Pull request by LeenMass**

### CODE REVIEW

**Review by MuhammadAw90 — DISMISSED** [review]

File: src/pages/Projects/index.jsx [hunk 1]
```diff
@@ -1,118 +1,68 @@
...
-              </div>
-            </div>
-            <button className={style.EditContainer}> <CiEdit className={style.innerIcons} /></button>
-            <button className={style.ArchiveContainer}> <BiArchiveIn className={style.innerIcons} /></button>
-          </div>
-        </div>
-      </div>
       <div className={style.export}>
-        <button className={style.PageButton}> Export as CSV </button>
+        <button> Export as CSV </button>
```
Comment (Karyum) [inline src/pages/Projects/index.jsx:42]: @majdya This can be a task later on

File: src/pages/Projects/index.jsx [hunk 2]
```diff
@@ -1,118 +1,68 @@
...
+import SearchBar from '../../components/SearchBar'
+import PageContainer from '../../components/PageContainer'
+import ContentsTable from '../../components/ContentsTable'
 
 const Projects = () => {
+  const [projects, setProjects] = useState([{}])
+  const getProjects = async () => {
+    try {
+      const result = await fetch('http://localhost:4000/Products')
+      const res = await result.json()
```
Comment (Karyum) [inline src/pages/Projects/index.jsx:13]: this is not critical, but the project does have [`Axios`](https://www.npmjs.com/package/axios), but fetch is also perfectly fine

File: src/components/SearchBar/index.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      <input
+        type="search"
+        className={style.searchbar}
+        placeholder="Search Project..."
+      />
+      <button className={style.srchico}> <BsSearch /></button>
+    </div>
+  )
+}
+export default SearchBar
```
Comment (Karyum) [inline src/components/SearchBar/index.jsx:21]: @majdya this can also be a task

File: src/components/PageContainer/index.jsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    <div className={style.container}>
+      <div>
+        <label className={style.MainLabel}>Projects</label>
+      </div>
+      <div>
+        <br />
+        <hr className={style.Line} />
+      </div>
+      {/* <div>{props.component}</div>
+      <div>{props.secondComponent}</div> */}
```
Comment (Karyum) [inline src/components/PageContainer/index.jsx]:
```suggestion

```

File: src/pages/Projects/index.jsx [hunk 5]
```diff
@@ -1,118 +1,68 @@
...
+// }
+
+// <Table
+// columns={[]}
+// data={[]}
+// onEdit={() => 1}
+// onArchive={() => 1}
+
+// className = "test"
+// />
```
Comment (Karyum) [inline src/pages/Projects/index.jsx]:
```suggestion

```

File: src/pages/Projects/index.jsx [hunk 6]
```diff
@@ -1,118 +1,68 @@
...
+import PageContainer from '../../components/PageContainer'
+import ContentsTable from '../../components/ContentsTable'
 
 const Projects = () => {
+  const [projects, setProjects] = useState([{}])
+  const getProjects = async () => {
+    try {
+      const result = await fetch('http://localhost:4000/Products')
+      const res = await result.json()
+      console.log(1, res)
```
Comment (Karyum) [inline src/pages/Projects/index.jsx]:
```suggestion

```

**Review by Karyum — DISMISSED** [review]
Great job here I don't see much to change just small things.
@majdya and @tawfiknasser I pointed out 2 comments that could be tasks later on

**Review by Karyum — APPROVED** [review]

**Pull request merged**